src/components/CreateDepartment.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CreateDepartmentComponent=()=>{
+    return(
+        <div>CREATE DEPARTMENT</div>
```

Comment:
only merge completed changes to dev. Changes under development can be kept in their respective feature branches.